src/main/resources/templates/ingredients/index.html

```diff
@@ -19,10 +18,11 @@
     </thead>
     <tr th:each="ingredient : ${ingredients}">
         <td th:text="${ingredient.name}"></td>
-        <td th:text="${ingredient.category.displayName}"></td>
-        <td th:text="${ingredient.toList.displayName}"></td>
+        <td th:text="${ingredient.category}"></td>
+        <td th:text="${ingredient.toList}"></td>
```

Comment:
This also needs .displayName for the enumeration value to appear in the table of ingredients